3.5/s2i/bin/run

```diff
@@ -15,6 +15,12 @@ function should_migrate() {
   is_django_installed && [[ -z "$DISABLE_MIGRATE" ]]
 }
 
+# Utility function to update PYTHONPATH and change cwd
+function change_cwd() {
+  export PYTHONPATH=${PYTHONPATH:-}:`pwd`
```

Comment:
If not set, the result would begin with colon, is this expected? Should there be rather ```export PYTHONPATH=${PYTHONPATH:+$PYTHONPATH:}:`pwd` ```?